### ISSUE TITLE
What is the goal of SingleFileZ compared to SingleFile?

### ISSUE THREAD

**Issue by user0022** [gildas-lormeau/SingleFileZ#67]
Hello! Excuse-me stupid question maybe!!

Well I am an average user, not very talented but with a little technical interest in computing as I use Linux and have learned some bases of html and css (very few but well...), which not everybody does.
Despite of it, I use the SingleFileZ addon and am pleased with it, but I am not sure at all of the goal of the zip transformation. What I see is that it takes less place on the disk, so as I like this feature I use SFZ. But is it made for something else? Security? Easier upload somewhere else? Reliability? Easier working on the file with some software?... 
The comment by Vik in https://addons.mozilla.org/fr/firefox/addon/singlefilez/reviews/?src=search saying "to save space" seems to indicate that the space is the advantage.

Maybe if it is the case, some users like me or with yet less knowledge, could happen to come accross the SingleFileZ addon page, and not install it, but if they had understood that it saves space on their disk, they would have. So maybe (suggestion) it could be a good thing to explain this interest of the addon compared to SingleFile, as it is just said that the page is zipped, and maybe not everybody knows what is the result of "zipping"...

**Comment by gildas-lormeau** [comment]
First the 2 extensions are separated mostly for practical reasons. When I "discovered" that an extension like SingleFile could produce HTML/ZIP files, I was tempted to add this feature in SingleFile itself, as an option. However, I quickly noticed it was a bit risky because the changes to apply in the code to support the legacy format and this new format would have been too large. So, I decided to fork SingleFile in order to deliver faster this feature without taking too much risks.

I think the zip format offer 2 advantages:
 - files are smaller
 - files can be unzipped in order to extract resources like the images or fonts more easily

I wish I could implement the encryption to protect pages with a password but the last time I tried, I was not able to produce valid zip files. I guess this would give another advantage. 

Note that SingleFileZ appears in the comparison table here: https://github.com/gildas-lormeau/SingleFile#singlefilez and the first question in the FAQ also explains the pros and cons: https://github.com/gildas-lormeau/SingleFileZ/blob/master/faq.md.

**Comment by user0022** [comment]
Thank you for clear answer! 

Nevertheless I still tell myself that maybe the advantage of page size reducing could me more explicitly indicated, as one can also imagine someone looking for this feature typing e. g. "reduce size" or "save space" or "smaller" or "lighter" in the Firefox addon search engine (or even in "a big general search engine"), which won't lead to this addon. But not thinking about typing "zip" which seems the only word in the page indicating this feature. (Previously there were tags in Firefox addon pages that could be used for this).

I should have clicked on the links that are in the addon page, the day I installed it! But too much in a hurry. It would have avoided an useless question but well it gave me also the opportunity to give my opinion about the other thing which is maybe not very useful too but well to me it seems to make sense. Now I watched it and it's instructive and answers my question yes.
